### DISCUSSION TITLE
Chest Auto Locking

### DISCUSSION THREAD

**Discussion by guerito52** [GriefPrevention/GriefPrevention#1891] [Q&A]
Is there a way to turn off auto locking of storage items? this is causing issues when players in my server place a shulker box. They place a shulker box and forget to unlock when they go and pick it up, and now they lost that shulker box with everything in it

**Comment by Jikoo** [comment]
GP does not prevent you from picking up shulker boxes in your own claims. It also only creates the new user claim specifically for normal untrapped chest placement, no other containers. Sounds like another plugin or a bad addon.

https://github.com/TechFortress/GriefPrevention/blob/d75b3cdd8fd51c67478bbf01ecfa0b78b8cdb41f/src/main/java/me/ryanhamshire/GriefPrevention/BlockEventHandler.java#L327

**Reply by guerito52** [reply]
Is it possible for storage items to auto lock by default?

**Reply by Jikoo** [reply]
You cannot configure additional storage items to automatically create a new player claim, no. As the claim creation is aimed at teaching new players about claims, it specifically targets the first container they are likely to place, a plain chest. Advanced users are expected to have their own claim creation tool and manage claim creation themselves. GP does warn when placing chests in unclaimed area, but it's on the user to do something with that. An addon could change that behavior.
